cmd/metrics-server/app/options/kubelet_client.go

```diff
@@ -72,6 +73,7 @@ func (o *KubeletClientOptions) Validate() []error {
 func (o *KubeletClientOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&o.InsecureKubeletTLS, "kubelet-insecure-tls", o.InsecureKubeletTLS, "Do not verify CA of serving certificates presented by Kubelets.  For testing purposes only.")
 	fs.BoolVar(&o.KubeletUseNodeStatusPort, "kubelet-use-node-status-port", o.KubeletUseNodeStatusPort, "Use the port in the node status. Takes precedence over --kubelet-port flag.")
+	fs.StringVar(&o.KubeletAnnotationPath, "kubelet-use-node-annotation-path", o.KubeletAnnotationPath, "Use this key in the node's annotations map to get the path for collecting metrics from Kubelets.")
```

Comment:
Could we maybe propose some sain default? Like `metrics.k8s.io/node-resource-metrics` <- just a guess on domain, please look up domain.

Reply:
I agree with you